airflow/providers/google/cloud/example_dags/example_automl_tables.py

```diff
@@ -139,7 +139,6 @@ def get_target_column_spec(columns_specs: List[Dict], column_name: str) -> str:
         task_id="update_dataset_task",
         dataset=update,
         location=GCP_AUTOML_LOCATION,
-        project_id=GCP_PROJECT_ID,
```

Comment:
Is this change necessary?

Reply:
This parameter was unused in `CloudAutoMLHook.update_dataset`, so I dropped the parameter from that method, as well as `AutoMLTablesUpdateDatasetOperator` and the example code here.

Reply:
Yeah. I thik it's good :)